utilities_common/multi_asic.py

```diff
@@ -0,0 +1,159 @@
+import subprocess
+import json
+import click
+import functools
+from sonic_device_util import get_num_npus
+from sonic_device_util import get_namespaces
+from sonic_device_util import is_multi_npu
+from sonic_device_util import is_port_internal
+from sonic_device_util import is_port_channel_internal
+from sonic_device_util import is_bgp_session_internal
+from sonic_device_util import get_all_namespaces
+
+import swsssdk
+from swsssdk import ConfigDBConnector
+import argparse
+
+DEFAULT_NAMESPACE = ''
+DISPLAY_ALL = 'all'
+DISPLAY_EXTERNAL = 'frontend'
+PORT_CHANNEL_OBJ = 'PORT_CHANNEL'
+PORT_OBJ = 'PORT'
+BGP_NEIGH_OBJ = 'BGP_NEIGH'
```

Comment:
wondering if some of these constants can potentially be used in other files. If that's the case lets create utilities_common/consts.py. It's cleaner and more readable if all constants are in one place.

Reply:
add a new file constants.py with all the constants in it